### ISSUE TITLE
msodde xml node check

### ISSUE THREAD

**Issue by n0way0ut** [decalage2/oletools#202]
I don't have the ssh key with me right now so I can't send a PR.
Please check if the elem.text is None before to add in text, like:

```
def process_file(filepath):
    z = zipfile.ZipFile(filepath)
    data = z.read('word/document.xml')
    z.close()
    # parse the XML data:
    root = ET.fromstring(data)
    text = u''
    # find all the tags 'w:instrText':
    # (each is a chunk of a DDE link)
    for elem in root.iter(TAG_W_INSTRTEXT):
        # concatenate the text of the field:
       	if elem.text is None:
	    continue
	text += elem.text
    return text
```

**Comment by thomasjfox** [comment]
Good catch :)

**Comment by n0way0ut** [comment]
it was simple, failed with the first doc that I tested :)

**Comment by decalage2** [comment]
Thanks, will fix it!

**Comment by decalage2** [comment]
By any chance, is it possible to share the sample that triggered the error?

**Comment by n0way0ut** [comment]
sure.
https://www.virustotal.com/en/file/f945105f5a0bc8ea0d62a28ee62883ffc14377b6abec2d0841e88935fd8902d3/analysis/

**Comment by decalage2** [comment]
Fixed in commit 9f71892c2534848e7dc693b61a4d03bae3e0fe34
